roles/stackdriver_logging/tasks/debian.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Stackdriver Logging | Debian | Add the Stackdriver repository
```

Comment:
_let me know if this is better reported in a new issue. i'm not sure if this feature is already "GA" so I didn't want to open one more issue_

This task should be executed after the next (apt_key) task. Right now the 0.0.10.1 release is failing to run this role because this task tries to update the apt cache with the new stackdriver repo but the GPG key is not known yet

This is the error
```
The full traceback is:
  File "/tmp/ansible_apt_repository_payload_l7nbst2w/ansible_apt_repository_payload.zip/ansible/modules/packaging/os/apt_repository.py", line 548, in main
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 586, in update
    raise FetchFailedException(e)
test-3 | FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "codename": null,
            "filename": "google-cloud-logging",
            "install_python_apt": true,
            "mode": null,
            "repo": "deb https://packages.cloud.google.com/apt google-cloud-logging-wheezy main",
            "state": "present",
            "update_cache": true,
            "validate_certs": true
        }
    },
    "msg": "apt cache update failed"
}
```
i was able to trace it to this error without ansible
```
sudo apt update
Hit:1 http://us-east1.gce.archive.ubuntu.com/ubuntu bionic InRelease
Get:2 http://us-east1.gce.archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]                                                                                                       
Get:3 http://us-east1.gce.archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]                                                                                                     
Get:4 https://packages.cloud.google.com/apt google-cloud-logging-wheezy InRelease [3801 B]                                                                                                   
Get:5 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]                                                                                 
Hit:6 http://archive.canonical.com/ubuntu bionic InRelease                      
Err:4 https://packages.cloud.google.com/apt google-cloud-logging-wheezy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6A030B21BA07F4FB
Reading package lists... Done
W: GPG error: https://packages.cloud.google.com/apt google-cloud-logging-wheezy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6A030B21BA07F4FB
E: The repository 'https://packages.cloud.google.com/apt google-cloud-logging-wheezy InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```